mim/commands/install.py

```diff
@@ -236,6 +236,11 @@ def infer_find_url(package: str) -> str:
     find_url = ''
     if package in WHEEL_URL:
         torch_v, cuda_v = get_torch_cuda_version()
+
+        # x.y.z -> x.y.0
```

Comment:
Add comments to explain why we do that. For example, indicating that there are usually no big changes between pytorch 1.8.1 and 1.8.0, thus the mmcv-full built in pytorch 1.8.0 works on both versions